app/models/form/subsection.rb

```diff
@@ -30,9 +30,10 @@ def status(case_log)
       return :cannot_start_yet
     end
 
-    qs = applicable_questions(case_log).reject { |q| CaseLog::OPTIONAL_FIELDS.include?(q.id) }
+    qs = applicable_questions(case_log)
+    qs_optional_removed = qs.reject { |q| CaseLog::OPTIONAL_FIELDS.include?(q.id) }
     return :not_started if qs.all? { |question| case_log[question.id].blank? || question.read_only? }
-    return :completed if qs.all? { |question| question.completed?(case_log) }
+    return :completed if qs_optional_removed.all? { |question| question.completed?(case_log) }
```

Comment:
is this just a refactor?

Reply:
No, it was to fix a bug with the property information section since line 34 meant that the subsection was considered not started even if you answered the first 3 questions of it (since they're included in the optional fields and filtered out)

Reply:
Ah, okay, gotcha!